srcpkgs/rclone/template

```diff
@@ -35,4 +36,9 @@ post_install() {
 	ln -sf rclone ${DESTDIR}/usr/bin/mount.rclone
```

Comment:
Just one more thing:

```suggestion
	ln -s rclone ${DESTDIR}/usr/bin/mount.rclone
```

https://github.com/void-linux/void-packages/pull/33884#discussion_r744033333

Reply:
done, thanks

Reply:
-sf is actually correct, we sometimes want to run one step twice and without -f, it would fail